k8s-tools/manage-cluster

```diff
@@ -354,24 +359,59 @@ END
 
   OR, you can source ${sourcing_file}
   "
-}
 
-function config_cluster() {
-  debug_log "config_cluster function"
+  if command -v helm >/dev/null; then
+    helm repo add crs4 https://crs4.github.io/helm-charts/
```

Comment:
This breaks if helm is installed but not initialized:

```
Error: Couldn't load repositories file (/home/simleo/.helm/repository/repositories.yaml).
You might need to run `helm init` (or `helm init --client-only` if tiller is already installed)
```

Reply:
Thanks for pointing out this problem.  I just fixed it with f379ae5